docs/pages/enroll-resources/server-access/guides/encrypted-session-recordings.mdx

```diff
@@ -0,0 +1,269 @@
+capturing a new recording and downloading the resulting `.tar` file from your
+[audit sessions backend](../../../reference/architecture/session-recording.mdx#storage).
+Replaying the session file directly with `tsh play` should fail and replaying
+using the session ID should succeed.
+
+## Manual Encryption Key Management
+
+<Admonition type="note">
+Manual encryption key management places all responsibility and complexity of
+managing keys on the administrator. This comes with much more risk of a
+misconfiguration impacting Teleport's ability to record or replay sessions.
+It's for this reason that Teleport Cloud does not allow use of manual
+encryption key management. Consider using the default automatic key management
+unless your usage explicitly requires manual key management.
+</Admonition>
+
+By default, Teleport will automatically provision and manage session recording
+encryption keys. However there may be cases where automatic management is not
+possible or desired, such as environments where Teleport lacks sufficient
+privileges to manage keys directly. For this reason, Teleport can be configured
+to use externally managed keys.
+
+### Configuration
+
+```yaml
+# snippet from session_recording_config
+encryption:
+  enabled: yes
+  manual_key_management:
+    enabled: yes
+    active_keys:
+      - type: pkcs11
+        label: 'session_recordings_002'
+    rotated_keys:
+      - type: pkcs11
+        label: 'session_recordings_001'
+```
+
+This example enables `manual_key_management` with one active key and one
+rotated key configured. Both expect a PKCS#11 compliant HSM as the key storage
+backend and each have their own label used to identify the keys within the HSM.
+
+Teleport will search the HSM for the keys described in both `active_keys` and
+`rotated_keys`. The active keys will be used during encryption and replay of
+any new session recordings. The rotated keys will be used during replay of
+historical session recordings, but will not be used during any encryption
+operations.
+
+As mentioned previously, it is important that all Teleport Auth Services
+have access to the same keys. Degraded availability of replays should be
+expected if unshared keys are used.
+
+<Admonition type="warning">
+Session recording encryption relies on a form of envelope encryption using OAEP
+with 4096-bit RSA key pairs. This means RSA 4096 is the only key type eligible
+for use with `manual_key_management`. It is also important that these keys are
+permitted to be used for decryption. What this means varies per key storage
+backend, but generally must be defined when the key is first provisioned.
+</Admonition>
+
+## Key rotation
+
+Session recording encryption keys are rotated using `tctl`. Rotation is a two
+phase process that requires initiating a rotation and then completing the
+rotation. It is also possible to rollback an in-progress rotation in the event
+that the previous key state needs to be restored.
+
+## Step 1/3. Initiate a rotation
```

Comment:
We generally don't want to include multiple sets of "Step..." H2s within a single guide, since this makes the structure a little convoluted. Would it make sense to split up this guide into its own subsection of the documentation, e.g., in `docs/pages/enroll-resources/server-access/encrypted-session-recordings/rotation.mdx`?